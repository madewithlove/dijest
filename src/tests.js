export {default as assert} from './tests/assert';
export {default as component} from './tests/component';
export {default as saga} from './tests/saga';
export {reducerTestFactory, default as reducer} from './tests/reducer';
