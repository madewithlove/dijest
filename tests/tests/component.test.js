import React, {Component} from 'react';
import {component} from '../../dist/tests';

const DummyComponent = props => <p {...props} />;

component('can test component', <DummyComponent foo="bar" />);
