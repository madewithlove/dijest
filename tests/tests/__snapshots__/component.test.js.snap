// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`can test component 1`] = `
<p
  foo="bar"
/>
`;
